fluentlenium-core/src/main/java/org/fluentlenium/core/wait/FluentWaitIntegerMatcher.java

```diff
@@ -0,0 +1,110 @@
+package org.fluentlenium.core.wait;
+
+import com.google.common.base.Predicate;
+import org.fluentlenium.core.Fluent;
+import org.fluentlenium.core.conditions.FluentListConditions;
+import org.fluentlenium.core.conditions.IntegerConditions;
+
+import static org.fluentlenium.core.wait.FluentWaitMessages.equalToMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.greaterThanMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.greaterThanOrEqualToMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.isPredicateNotVerifiedMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.isPredicateVerifiedMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.lessThanMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.lessThanOrEqualToMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.notEqualToMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.notGreaterThanMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.notGreaterThanOrEqualToMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.notLessThanMessage;
+import static org.fluentlenium.core.wait.FluentWaitMessages.notLessThanOrEqualToMessage;
+
+
+/**
+ * Matcher used for integers like elements size
+ */
+public class FluentWaitIntegerMatcher implements IntegerConditions {
+    private final AbstractWaitElementListMatcher matcher;
+
+    public FluentWaitIntegerMatcher(AbstractWaitElementListMatcher matcher) {
+        this.matcher = matcher;
```

Comment:
In this case, it's mandatory to use this because local variable and class variable have the same name (matcher).